journal/src/main/scala/com/evolutiongaming/kafka/journal/ConsumeActionRecords.scala

```diff
@@ -18,10 +22,44 @@ trait ConsumeActionRecords[F[_]] {
 
 object ConsumeActionRecords {
 
+  @deprecated("use `of` instead", "2023-07-26")
   def apply[F[_] : BracketThrowable](
     consumer: Resource[F, Journals.Consumer[F]],
     log: Log[F])(implicit
     consRecordToActionRecord: ConsRecordToActionRecord[F]
+  ): ConsumeActionRecords[F] = apply1(_ => consumer, log)
+
+  def of[F[_] : Concurrent : Clock](
+    consumerPool: ResourcePool[F, Journals.Consumer[F]],
```

Comment:
why not use `Resource[F, Journals.Consumer[F]]` ?

Reply:
why this class need to know anything about underlying pool and even combine metrics? 
I think you should create separate class that takes `metrics/consumer/pool settings` and returns `Resource[consumer]`

Reply:
>  create separate class that takes metrics/consumer/pool settings and returns Resource[consumer]
This would be nice, but the problem is I want `Topic` for writing metrics and `Topic` comes from `Key`, so I don't know `Topic` in advance when constructing this pooled consumer. Or I have to have a complicated signature like
`def pooledConsumer[F[_]](...): Resource[F, Topic => Resource[F, Consumer[F]]]`

Reply:
I guess it's ok if we don't add topic to consumer pool metrics. Created `PooledConsumer` object.